src/Language/GraphQL/Draft/Parser.hs

```diff
@@ -479,3 +489,41 @@ between open close p = tok open *> p <* tok close
 -- `empty` /= `pure mempty` for `Parser`.
 optempty :: Monoid a => Parser a -> Parser a
 optempty = option mempty
+
+-- | Parses strings delimited by triple quotes. 
+-- http://spec.graphql.org/June2018/#sec-String-Value
+blockString :: Parser Text
+blockString = do
+  _ <- triplequotes <?> "opening triple quotes"
+  str <- T.lines . T.pack <$> AT.manyTill AT.anyChar triplequotes <?> "the body of a triple quoted string"
+  let !cleanLines = dropEmptyLines str
+  let smallest = foldr selectSmallest maxBound cleanLines -- counts indentation and selects the smallest one from all lines
+  let fixedLines = foldr (fixIndentation smallest) "" cleanLines -- removes the common indentation from all lines.
+  return (T.strip . toStrict . TB.toLazyText $ fixedLines)
+ where
+  -- used to remove the common indentation from each line.
+  fixIndentation :: Int -> Text -> TB.Builder -> TB.Builder 
+  fixIndentation smallest a acc =
```

Comment:
Ditto, I think it would be cleaner to stick to the existing indentation style.

Reply:
```suggestion
  let fixedLines = foldr (fixIndentation smallest) "" cleanLines -- removes the common indentation from all lines.
  return (T.strip . toStrict . TB.toLazyText $ fixedLines)
 where
  -- used to remove the common indentation from each line.
  dropIndentation :: Int -> Text -> TB.Builder -> TB.Builder 
  dropIndentation smallest a acc =
```

Reply:
I don't think my first comment above has been resolved.

Reply:
I'll take a look at it